pages/design/atomic-template-empty.md

```diff
@@ -0,0 +1,82 @@
+---
+name:
+test_type: atomic
+
+description: |
+  This rule checks ...
+
+success_criterion:
+- x.x.x
+
+test_aspects:
+- 
+
+authors:
+-
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any (??) element ...
+
+### Expectation (1)
+
+Each target element ...
+
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility support
+
+*There are no major accessibility support issues known for this rule.*
+
+## Background
+
+- 
```

Comment:
Add "(e.g. WCAG Techniques or links with background information mentioned in Applicability, Expectations or Assumptions)" ?